scripts/utils.sh

```diff
@@ -0,0 +1,55 @@
+#/usr/bin/env bash
+
+#######################################
+# Common setup. Init MacOS compatibility
+# variables.
+# Globals:
+#   READLINK
+#######################################
+function common_setup
```

Comment:
Why does this function need to be called explicitly to set up `$READLINK`? Can't the body just be executed when the script is sourced?

Reply:
True. I just decided that everything in this file should be a function. I think that makes things cleaner